apps/trip_plan/lib/trip_plan/api/open_trip_planner/builder.ex

```diff
@@ -1,26 +1,40 @@
 defmodule TripPlan.Api.OpenTripPlanner.Builder do
   alias TripPlan.Api.OpenTripPlanner, as: OTP
+  alias TripPlan.NamedPosition
   alias Util.{Polygon, Position}
   @ten_miles 16_093
 
+  @spec build_query_params(Position.t(), Position.t(), TripPlan.Api.plan_opts()) ::
+          {:ok, %{String.t() => String.t()}} | {:error, any}
+  def build_query_params(from, to, opts) do
+    from_string = location(from)
+    to_string = location(to)
+    default_mode_string = "[{mode: WALK}, {mode: TRANSIT}]"
+
+    do_build_params(opts, %{
+      "fromPlace" => from_string,
+      "toPlace" => to_string,
+      "transportModes" => default_mode_string,
+      "walkReluctance" => 5,
```

Comment:
**suggestion**: The results are super different with no walk distance configured. After playing around with various itineraries it looks like bumping up this value can compensate for that. I think `15` produced a nice result.

Reply:
though if you increase that you'll want to increase the value in line 74 (for `{:optimize_for, :less_walking}`) to be even greater!

I also recognize these are totally subjective things, but in this case I was aiming for comparable results to current state.